packages/edit-site/src/components/dataviews-actions/index.js

```diff
@@ -41,3 +41,33 @@ export const useEditPostAction = () => {
 		[ history ]
 	);
 };
+
+export const useQuickEditPostAction = () => {
+	const history = useHistory();
+	return useMemo(
+		() => ( {
+			id: 'quick-edit-post',
+			label: __( 'Quick Edit' ),
```

Comment:
See this [comment](https://github.com/WordPress/gutenberg/pull/63600#issuecomment-2286037002) for a different label..

Reply:
Another option (common across similar features in other site builders) would be something like "Page settings".